tools/appliance/systemvmtemplate/template-base_x86_64-target_aarch64.json

```diff
@@ -0,0 +1,91 @@
+{
+  "_license": "Apache License 2.0",
+  "builders": [
+    {
+      "boot_command": [
+        "c<wait>",
+        "linux /install.a64/vmlinuz <wait>",
+        "preseed/url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/preseed_aarch64.cfg <wait>",
+        "debian-installer=en_US.UTF-8 <wait>",
+        "auto <wait>",
+        "language=en locale=en_US.UTF-8 <wait>",
+        "kbd-chooser/method=us <wait>",
+        "keyboard-configuration/xkb-keymap=us <wait>",
+        "netcfg/get_hostname=systemvm <wait>",
+        "netcfg/get_domain=apache.org <wait>",
+        "country=IN keymap=us <wait>",
+        "fb=false <wait>",
+        "debconf/frontend=noninteractive <wait>",
+        "console-setup/ask_detect=false <wait>",
+        "console-keymaps-at/keymap=us <wait>",
+	"---",
+        "<enter><wait>",
+	"initrd /install.a64/initrd.gz",
+	"<enter><wait>",
+	"boot<enter><wait>"
+      ],
+      "boot_wait": "60s",
+      "disk_interface": "virtio",
+      "cdrom_interface": "virtio-scsi",
+      "disk_size": "5000M",
+      "format": "qcow2",
+      "headless": true,
+      "http_directory": "http",
+      "iso_checksum": "sha512:4980197fa30646a43df2672d98500a182d6636422e39e809d986551356f4d24b1ffab6d745331369b27f2e9a5326f020ba141df5772ac2c6328c08cce73de7e8",
+      "iso_url": "https://cdimage.debian.org/mirror/cdimage/archive/11.10.0/arm64/iso-cd/debian-11.10.0-arm64-netinst.iso",
+      "net_device": "virtio-net",
+      "output_directory": "../dist",
+      "qemu_binary": "qemu-system-aarch64",
```

Comment:
@StepBee had you tested aarch64 systemvmtemplate build on x86_64 host? I'm trying to automate this with a Jenkins build and getting this:
```
10:56:47 2024/07/24 10:56:47 packer-builder-qemu plugin: Executing /usr/bin/qemu-system-aarch64: []string{"-netdev", "user,id=user.0,hostfwd=tcp::2680-:22", "-machine", "virt", "-cpu", "cortex-a72", "-pflash", "/usr/share/AAVMF/AAVMF_CODE.fd", "-boot", "strict=off", "-drive", "file=../dist/systemvmtemplate,if=virtio,cache=writeback,discard=ignore,format=qcow2", "-drive", "file=/var/lib/jenkins/.cache/packer/f625f9a7fa1737d7bc5cd5e20e9ca14107ead36b.iso,if=none,index=0,id=cdrom0,media=cdrom", "-name", "systemvmtemplate", "-m", "500M", "-smp", "1", "-monitor", "none", "-vnc", "127.0.0.1:95", "-device", "virtio-net,netdev=user.0", "-device", "virtio-scsi-pci,id=scsi0", "-device", "virtio-scsi-device", "-device", "scsi-cd,drive=cdrom0"}
10:56:47 2024/07/24 10:56:47 packer-builder-qemu plugin: Started Qemu. Pid: 125333
10:56:47 2024/07/24 10:56:47 packer-builder-qemu plugin: Qemu stderr: WARNING: Image format was not specified for '/usr/share/AAVMF/AAVMF_CODE.fd' and probing guessed raw.
10:56:47 2024/07/24 10:56:47 packer-builder-qemu plugin: Qemu stderr:          Automatically detecting the format is dangerous for raw images, write operations on block 0 will be restricted.
10:56:47 2024/07/24 10:56:47 packer-builder-qemu plugin: Qemu stderr:          Specify the 'raw' format explicitly to remove the restrictions.
10:56:47 2024/07/24 10:56:47 packer-builder-qemu plugin: Qemu stderr: qemu-system-aarch64: Initialization of device cfi.pflash01 failed: Could not reopen file: Permission denied
10:56:47 [1;31m==> qemu: Error launching VM: Qemu failed to start.
```

Host in Ubuntu 22.04 x86_64 VM (same one used to build the x86-64 systemvmtemplates) cc @kiranchavala @NuxRo

Reply:
@rohityadavcloud yes indeed, i mostly build aarch64 systemvmtemplate on x86_64 host because of better performance compared to building on raspi or the likes of.

My host is running Ubuntu 20.04.6 LTS x86_VM.
Latest systemvm i build it for is 4.19.01

Reply:
Great, many thanks @StepBee

Reply:
@StepBee for reference, this is the full Jenkins job log:
[jenkins-aarch64-job.txt](https://github.com/user-attachments/files/16357740/jenkins-aarch64-job.txt)

Reply:
Sorry double checked, my host is:
Ubuntu 20.04.6 LTS VM

Reply:
So we are on the same OS and same Version.

I am executing the build process as root user, where i see jenkins is using user ubuntu

To check we are using the same file
```
# ls -al /usr/share/AAVMF/AAVMF_CODE.fd
-rw-r--r-- 1 root root 67108864 Feb 14 00:52 /usr/share/AAVMF/AAVMF_CODE.fd
# md5sum /usr/share/AAVMF/AAVMF_CODE.fd
9a385b6b7377ea8c3fad4a3ad72564a3  /usr/share/AAVMF/AAVMF_CODE.fd
```

Could you try to copy AAVMF_CODE.fd to a second file and refer to it in a second pflash parameter in the template json like
`cp /usr/share/AAVMF/AAVMF_CODE.fd /usr/share/AAVMF/AAVMF_CODE_2.fd`
and in
template-base_x86_64-target_aarch64.json
below the existing line:
`[ "-pflash", "/usr/share/AAVMF/AAVMF_CODE.fd" ],`
you add a second pflash line like:
`[ "-pflash", "/usr/share/AAVMF/AAVMF_CODE_2.fd" ],`

and see if this actually makes a difference?

Reply:
@StepBee using the -bios flag worked for me.